bundles/org.openhab.binding.unifi/README.md

```diff
@@ -103,6 +104,15 @@ The following table describes the `poePort` configuration parameters:
 | portNumber | The port number as reported by the switch (starts with 1) | Required |
 | macAddress | The MAC address of the switch device the port is part of  | Required |
 
+### `accessPoint`
+
+The following table describes the `accessPoint` configuration parameters:
+
+| Parameter    | Description                               | Config   | Default |
+| ------------ | ------------------------------------------|--------- | ------- |
+| cid          | The MAC address of the access point       | Required | -       |
+| site         | The site where the client should be found | Optional | -       |
```

Comment:
Should it be "access point" rather than "client"?
```suggestion
| site         | The site where the client should be found | Optional | -       |
```